src/commons/BrowserUtils.ts

```diff
@@ -706,6 +708,36 @@ export namespace BrowserUtils {
     }
   }
 
+  /**
+   * Change size of browser window
+   *    If only one parameter is provided the second will stay as is.
+   *    If no parameter provided there will be no change.
+   * @param requestedWidth - Width (px)
+   * @param requestedHeight - Height (px)
+   */
+  export function setWindowSize(
+    requestedWidth?: number,
```

Comment:
need better api here.
maybe something like:  setWindwoSize({width:val, height:val})
this way you want see calls like:         BrowserUtils.setWindowSize(undefined, 768);
the undefined is very strange

Reply:
done